server/src/main/java/org/elasticsearch/bootstrap/Elasticsearch.java

```diff
@@ -75,18 +76,18 @@ private static void exitWithUserException(PrintStream err, int exitCode, Excepti
         gracefullyExit(err, exitCode);
     }
 
-    private static void exitWithUnknownException(PrintStream err, Exception e) {
-        if (System.getProperty("es.logs.base_path") != null) {
-            // this is a horrible hack to see if logging has been initialized
-            // we need to find a better way!
-            Logger logger = LogManager.getLogger(Elasticsearch.class);
-            logger.error("fatal exception while booting Elasticsearch", e);
-        }
-        // format exceptions to the console in a special way to avoid 2MB stacktraces from guice, etc.
-        StartupException.printStackTrace(e, err);
+    private static void exitWithUnknownException(PrintStream err, Throwable e) {
+        Logger logger = LogManager.getLogger(Elasticsearch.class);
+        logger.error("fatal exception while booting Elasticsearch", e);
```

Comment:
so given we use %consoleException for console appender this will result with `Elasticsearch failed to startup normal...` in console.
However we will print full stacktrace in server logs now? Do we want this?

Reply:
I am worried that this is harder to control. THe exception formatting in json logs is controlled by ecs-logging-lib and I don't see a way to limit the size of stactrace..
https://github.com/elastic/ecs-logging-java/blob/723a5572eb9f9f462605df2cf7a601571c4e6dee/ecs-logging-core/src/main/java/co/elastic/logging/EcsJsonSerializer.java#L239